tmt/steps/prepare/feature/fips.py

```diff
@@ -0,0 +1,32 @@
+import dataclasses
+from typing import Any, Optional
+
+import tmt.log
+import tmt.steps.prepare
+import tmt.utils
+from tmt.steps.prepare.feature import Feature, PrepareFeatureData, provides_feature
+from tmt.steps.provision import Guest
+from tmt.utils import field
+
+
+@dataclasses.dataclass
+class FipsStepData(PrepareFeatureData):
+    fips: Optional[str] = field(
+        default='enabled',
+        option='--fips',
+        metavar='enabled',
+        help='Whether FIPS mode should be enabled')
+
+
+@provides_feature('fips')
+class Fips(Feature):
```

Comment:
@happz is there a way to inject there more user facing documentation?

Reply:
Yes, plugin can have a docstring, and that would be rendered in docs (the PR adding `prepare/feature` plugin docs to Sphinx). We don't have a docstring in the EPEL plugin, but we should - maybe we could leave it be here, merge this PR and the one rendering docs, and tackle both plugins at the same time.